### ISSUE TITLE
Yosemite: "verifying" forever

### ISSUE THREAD

**Issue by demisjohn** [Kazzz-S/klayout#37]
Hello @Kazzz-S ,
Thanks for all your work to keep KLayout installers work for MacOS!

I was trying to have my coworker learn KLayout, but found that, on her iMac running Yosemite (10.10.5), the old Yosemite-compatible versions of klayout (0.25.9) never launch, instead MacOS shows a "verifying" progress bar that never ends as far as we can tell.

Any suggestions on how to get KLayout running on her computer?
In the mean time I will find a newer laptop and try to get her running on that.

(And this has obviously made it clear that we do need to upgrade her computer, as it's actually hindering our work now!)
Thanks.

**Comment by Kazzz-S** [comment]
Hello @demisjohn,

Thank you for using the MacOS packages.
Is she facing the issue just after the initial installation?
I do not remember such trouble on Yosemite.
My machine is MacBook Pro Mid 2012 model with 16GB main memory.
Does her machine have enough memory?

One thing I can suggest is:
  1.  rename (or delete) $HOME/.klayout/klayoutrc 
  2. restart the tool

After releasing the official DMG package (0.25.9) to the community, there were some upgrades before 0.26.0.
Right now, I'm building the very last revision of 0.25.9 on Yosemite.
Once it's ready, I'll upload it to my Dropbox.

Thanks
Kazzz

**Comment by Kazzz-S** [comment]
Hello @demisjohn,

A new 0.25.9 build **klayout-0.25.9-MacOSX-Yosemite-4-Qt5101mp.dmg** is available under:
https://www.dropbox.com/sh/a7pmxxq01ua2j0h/AABjZpyzU8vvuca1957gpZRna?dl=0

Try this one if you like.
Regards,
Kazzz

**Comment by demisjohn** [comment]
Thanks, we’ll give it a shot!
Will let you know later today. 
-- Demis

**Comment by demisjohn** [comment]
Oh, I didn’t answer your questions below:
1) correct, we launch a brand KLayout after coping to our Applications folder, and it pops up a MacOS “verifying” progress bar. I believe this is the MacOS app verification, to ensure apps aren’t hacked or contain viruses etc. - I don’t know how it does that though. 
Disabling App verification (I found some “defaults write LaunchControl ....” CLI somewhere) did not help, but the “verifying” progress and did NOT appear, however the KLayout icon just bounced in the Dock forever and never launch (well, we gave it ~2-3 min and then gave up).

-- Demis

(Swnt frm my oPhone)
